core/src/main/java/hudson/lifecycle/Lifecycle.java

```diff
@@ -120,13 +120,13 @@ public void verifyRestartable() throws RestartNotSupportedException {
                     // if run on Unix, we can do restart
                     try {
                         instance = new UnixLifecycle();
-                    } catch (final IOException e) {
-                        LOGGER.log(Level.WARNING, "Failed to install embedded lifecycle implementation", e);
+                    } catch (final Throwable t) {
```

Comment:
Dead code, since this constructor could never in fact throw `IOException`, but I thought it best to keep this fallback since any unanticipated errors here should not be fatal.